crates/rolldown/src/bundle/bundle.rs

```diff
@@ -150,17 +151,15 @@ impl Bundle {
     let dist_dir = self.options.cwd.join(&self.options.out_dir);
 
     if self.options.clean_dir && self.options.dir.is_some() {
-      clean_dir(&self.fs, &dist_dir).map_err(|err| {
-        anyhow::anyhow!("Could not clean directory for output chunks: {}", dist_dir.display())
-          .context(err)
+      clean_dir(&self.fs, &dist_dir).with_context(|| {
+        format!("Could not clean directory for output chunks: {}", dist_dir.display())
       })?;
```

Comment:
I think these were written in the opposite order.
https://docs.rs/anyhow/latest/anyhow/trait.Context.html#example

Before:
```
Error: Read only (os error 0000)

Caused by:
    Could not clean directory for output chunks: ./path/to/dir
```
After:
```
Error: Could not clean directory for output chunks: ./path/to/dir

Caused by:
    Read only (os error 0000)
```